libs/protocol-client/src/lib/client/contribution.ts

```diff
@@ -241,6 +241,58 @@ export class Contribution extends BaseClient {
     });
   }
 
+  public async bulkAttest(
+    networkConfig: NetworkConfig,
+    signer: ethers.Signer,
+    chainId: number,
+    userId: number,
+    data: {
+      contribution: number;
+      confidenceId: number;
+      confidenceName: string;
```

Comment:
I've to pass confidence's name & id because it's required for create on chain attestation.